src/stores/useGovStore.ts

```diff
@@ -62,5 +83,8 @@ export const useGovStore = defineStore('govStore', {
     async fetchProposalVotes(proposalId: string, next_key?: string) {
       return this.blockchain.rpc.getGovProposalVotes(proposalId, next_key);
     },
+    async fetchProposalVotesVoter(proposalId: string, voter: string) {
```

Comment:
add it to Gov Store

Reply:
It looks like it's already in the govStore